atlasdb-config/src/main/java/com/palantir/atlasdb/factory/ServiceDiscoveringAtlasSupplier.java

```diff
@@ -68,7 +71,7 @@ public KeyValueService getKeyValueService() {
     }
 
     public synchronized TimestampService getTimestampService() {
-        DebugLogger.logger.info("Fetching timestamp service from thread {}. This should only happen once.",
+        log.info("Fetching timestamp service from thread {}. This should only happen once.",
```

Comment:
The below error messages will probably be noticed in the normal logs anyway?

If we wanted to keep using the `DebugLogger` here, one (hacky) option open to us is to keep `DebugLogger` in `com.palantir.timestamp`, but have it in the `atlasdb-commons` project...